assets/sass/application.sass

```diff
@@ -5,6 +5,7 @@ $path: "/assets/images/"
 
 @import 'govuk/all'
 @import 'moj/all'
+@import "node_modules/@ministryofjustice/hmpps-digital-prison-reporting-frontend/dpr/all"
```

Comment:
is it possible to add a load-path to the `bin/build-css.sh` so we don't need to reference the file structure here?